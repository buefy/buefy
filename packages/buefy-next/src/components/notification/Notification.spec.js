import { shallowMount } from '@vue/test-utils'
import BNotification from '@components/notification/Notification'

let wrapper

describe('BNotification', () => {
    beforeEach(() => {
        wrapper = shallowMount(BNotification)
    })

    it('is called', () => {
        expect(wrapper.vm).toBeTruthy()
        expect(wrapper.vm.$options.name).toBe('BNotification')
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
