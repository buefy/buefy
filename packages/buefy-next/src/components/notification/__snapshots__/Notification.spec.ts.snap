// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BNotification > render correctly 1`] = `
"<transition-stub name=\\"fade\\" appear=\\"false\\" persisted=\\"true\\" css=\\"true\\">
  <article class=\\"notification\\"><button class=\\"delete\\" type=\\"button\\"></button>
    <!--v-if-->
    <!--v-if-->
  </article>
</transition-stub>"
`;
